src/comgr.cpp

```diff
@@ -38,7 +38,7 @@
 #include <miopen/solver/implicitgemm_util.hpp>
 #include <miopen/stringutils.hpp>
 
-#if HIP_PACKAGE_VERSION_FLAT >= 5004000000ULL
+#if !defined(_WIN32) && (HIP_PACKAGE_VERSION_FLAT >= 5004000000ULL)
```

Comment:
Why in Windows the location of amd_comgr.h does not correspond to the one in Linux? Is it a bug?